packages/valory/skills/transaction_settlement_abci/tests/test_rounds.py

```diff
@@ -107,6 +107,14 @@ def get_participants() -> FrozenSet[str]:
     return frozenset([f"agent_{i}" for i in range(MAX_PARTICIPANTS)])
 
 
+def get_uniform_payloads(round_cls, **kwargs):  # AbstractRound
+    """Get uniform payloads"""
+
+    payloads = {p: round_cls.payload(p, **kwargs) for p in get_participants()}
+    most_voted_content = next(iter(payloads.values())).data
+    return payloads, most_voted_content
+
+
 def get_participant_to_randomness(
```

Comment:
all these other helper functions can be removed later on, I left them for now since they are used in the synchronized data test still (only) and removing and changing that would introduce unnecessarily many changes in this PR.